strings/src/main/res/values/strings.xml

```diff
@@ -254,7 +254,6 @@
     <string name="selected_invalid_image">Selected file is not a valid image</string>
     <string name="take_picture_instruction">Tap the screen to take a picture</string>
     <string name="error_front_camera_unavailable">Front camera not available on this device</string>
-    <string name="camera_error">This device doesn\'t support the Camera2 API.</string>
```

Comment:
This shouldn't be a possibility any longer as ["CameraX supports devices running [Android 5.0 (API level 21)](https://developer.android.com/about/versions/lollipop) and higher,"](https://developer.android.com/training/camerax).